src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <>
```

Comment:
redundant [react fragment](https://react.dev/reference/react/Fragment) - you can read more about usage